leapyear.js

```diff
@@ -1,9 +1,3 @@
-/**
- * @param {number} year - year
- * @return {boolean} returns true if the given year is leap year
-   otherwise it returns false 
- */
-
 function checkLeapyear(year) { 

```

Comment:
/**
 * Check Leap year or not
 * @param {number} year - year
 * @returns {boolean} - returns true if it is leap year
 */